tests/is/L42/perftests/PerfResult.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * 
+ */
+package is.L42.perftests;
+
+import is.L42.top.CachedTop;
+
+/**
+ * @author Claire
+ */
+public record PerfResult(PerfStat stat, CachedTop cache) {
```

Comment:
This may be the nested class of an appropriate class, to avoid a file for just one line!